fe/fe-core/src/main/java/org/apache/doris/analysis/SlotDescriptor.java

```diff
@@ -162,6 +162,21 @@ public void setIsMaterialized(boolean value) {
         isMaterialized = value;
     }
 
+    public void materializeSrcExpr() {
+        if (sourceExprs == null) {
+            return;
+        }
+        for (Expr expr : sourceExprs) {
```

Comment:
should we need to do chain replace? such as set `IsMaterialized` on the source expr of source expr's slot

Reply:
Yes, we do need